samples/language/Qubit.qs

```diff
@@ -0,0 +1,40 @@
+/// # Sample
+/// Qubit
+///
+/// # Description
+/// Q# uses the `Qubit` primitive type to represent quantum state. A qubit represents 
+/// the smallest addressable physical unit in a quantum computer. Qubits are long-lived,
+/// and accumulates transformations to quantum states. A Q# program has no ability to
+/// introspect into the state of a qubit, and thus is entirely agnostic about what a
+/// quantum state is or on how it is realized. Rather, a program can call operations
+/// such as Measure to learn information about the quantum state of the computation.
+namespace MyQuantumApp {
+    
```

Comment:
nit: I think the pattern so far has been to not have blank lines between the start of a namespace and `open` directives.

We need a linter to do all this for us!